src/App.css

```diff
@@ -46,12 +50,59 @@ tr {
 .results {
   padding-top: 15px;
 }
+.card {
+  float: left;
+  width: 30%;
+  height: 30rem;
+  padding: 10px;
+  margin: 20px;
+  border-style: solid;
+  border-color: black;
+}
+.card-img-top {
+  height: 10rem;
+}
 
 .footer {
+  clear: both;
+  position: relative;
+  width: 100%;
+  background-color: rgb(0, 0, 0);
+  color: white;
+  text-align: left;
+  height: 100px;
+  margin-top: -200px;
   padding-top: 20px;
-  text-align: center;
+}
+.card:hover {
+  opacity: 2;
+  transform: scale(1.05);
+  box-shadow: 2px 2px 10px 14px rgba(255, 255, 255, 0.2);
+  background-color: rgb(14, 165, 21);
+}
```

Comment:
Nice use of on hover properties here! With opacity, the docs state the value can be **A <number> in the range 0.0 to 1.0, inclusive, or a <percentage> in the range 0% to 100%** so will need to have a look at your value for this :)